src/MoBi.Presentation/Tasks/Edit/EditTaskForContainer.cs

```diff
@@ -86,6 +94,66 @@ public override void Save(IContainer container)
          exportContainer(container, fileName);
       }
 
+      private void addIndividualParametersToContainers(IndividualBuildingBlock individual, IContainer container)
+      {
+         if (individual == null || !individual.Any())
+            return;
+
+         var allSubContainers = container.GetAllContainersAndSelf<IContainer>();
+         allSubContainers.Each(subContainer =>
+         {
+            var subContainerPath = container.ParentPath.Concat(_objectPathFactory.CreateAbsoluteObjectPath(subContainer));
+            addIndividualParametersToContainer(individual, subContainer, new ObjectPath(subContainerPath));
+         });
+      }
+
+      private void addIndividualParametersToContainer(IndividualBuildingBlock individual, IContainer container, ObjectPath containerPath)
+      {
+         var individualParametersToExport = individual.Where(individualParameter => individualParameter.ContainerPath.StartsWith(containerPath)).ToList();
+
+         // Ensure that the distributed parameters are added to the container first. After all distributed parameters are added, then the other parameters can be added.
+         individualParametersToExport.Where(x => x.DistributionType != null).Each(x => addIndividualParameterToContainer(x, container, containerPath));
```

Comment:
shouldn't we better use `.Where(x => !x.IsDistributed())` instead?

Reply:
That does not yet exist for ```PathAndValueEntity``` based entities. I'll add the extension in Core, then add a core update here to use it after you approve the core PR